packages/suite/src/middlewares/suite/redirectMiddleware.ts

```diff
@@ -65,7 +65,7 @@ const redirect =
         switch (action.type) {
             // todo: this will not get call after acquiring device!
```

Comment:
@mroz22 Is this 'todo' valid? Maybe I don't get it, but it seems to me called after acquiring the device too.